### PR TITLE
Sprint 1/issue 12#build the home screen and navigation bar

### PR DESCRIPTION
**Pull request by STR1ND3R**
Added the navbar and the home screen with its controller

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  def index
+
```
Comment (MauricioNC) [inline app/controllers/home_controller.rb]: Remove this empty line

File: app/views/events/index.html.erb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <h1>Events lists</h1>
-<div>
+<div class="">
   <%= link_to "Create new event", new_event_path %>
 </div>
 
 <% @events.each_with_index do |event, index| %>    
   <h2><%= "Event no. #{format("%02d", (index + 1).to_s)} #{event.title}" %></h2>
-  <div><%= link_to 'Show event details', event_path(event) %></div>
+  <div class=""><%= link_to 'Show event details', event_path(event) %></div>
```
Comment (MauricioNC) [inline app/views/events/index.html.erb]: If you won't add any class, just remove the property

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -1,13 +1,8 @@
+<div class="w-screen flex justify-center mt-24">
+<h1>
+    Events managment system
+</h1>
+</div>
```
Comment (MauricioNC) [inline app/views/home/index.html.erb]: Indent correctly this part of your code

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -12,6 +12,16 @@
   </head>
 
   <body class="prose">
+
+    <nav class="bg-black h-28 w-screen flex flex-row ">
+      <div class="mx-4 py-5">
+        <%= image_tag("logo.png", class:"my-0 h-4/5") %>
+      </div>
+      <div class="mx-4 py-5">
+        <%= button_to "GetStarted", "events#index" , method: :get, class:"bg-black rounded py-5 px-2 text-white font-bold"%>
```
Comment (MauricioNC) [inline app/views/layouts/application.html.erb]: Don't user button_to for get requests, instead use link_to.
Use button_to only for delete and post requests

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -12,6 +12,16 @@
...
+
+      </div>
+    </nav>
```
Comment (MauricioNC) [inline app/views/layouts/application.html.erb]: You can export the nav in a separate file

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/views/home/index.html.erb (hunk 3 above)
Comment (STR1ND3R) [reply app/views/home/index.html.erb]: done

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (STR1ND3R) [reply app/views/layouts/application.html.erb]: done

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (STR1ND3R) [reply app/views/layouts/application.html.erb]: done

File: app/views/events/edit.html.erb [hunk 6]
```diff
@@ -1,3 +1,5 @@
 <h1>Edit event</h1>
+
 <%= render 'form', event: @event %>
-<%= link_to 'Back', event_path(@event), class: 'btn-3 ml-5 py-1 no-underline' %>
+
+<%= link_to 'Back', event_path(@event), class: 'btn-3 ml-5 py-1 no-underline' %>
```
Comment (MauricioNC) [inline app/views/events/edit.html.erb]: Add a new empty line

File: app/views/home/index.html.erb [hunk 7]
```diff
@@ -1,10 +1,7 @@
...
-  </ul>
-</nav>
-
+<div class="w-screen flex justify-center mt-24">
+  
 <h1>
   Events management system
 </h1>
+  
+</div>
```
Comment (MauricioNC) [inline app/views/home/index.html.erb]: Add a new empty line

File: app/views/layouts/_navbar.html.erb [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<nav class="bg-black h-28 w-screen flex flex-row ">
+  <div class="mx-4 py-5">
+    <%= image_tag("logo.png", class:"my-0 h-4/5") %>
+  </div>
+  <div class="mx-4 py-5">
+    <%= link_to "GetStarted", "events#index", class:"bg-black rounded py-5 px-2 text-white font-bold"%>
+
+  </div>
+</nav>
```
Comment (MauricioNC) [inline app/views/layouts/_navbar.html.erb]: Add a new empty line

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -12,6 +12,7 @@
   </head>
 
   <body class="prose">
+    <%= render 'layouts/navbar' %>
```
Comment (MauricioNC) [inline app/views/layouts/application.html.erb]: You can add the navbar in shared folder like:

/views/shared/_navbar.html.erb

And render like: 

<%= render 'shared/navbar' %>

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/views/home/index.html.erb (hunk 7 above)
Comment (juperameza) [inline app/views/home/index.html.erb]: New line

File: app/views/events/edit.html.erb (hunk 6 above)
Comment (STR1ND3R) [reply app/views/events/edit.html.erb]: done

File: app/views/layouts/_navbar.html.erb (hunk 8 above)
Comment (STR1ND3R) [reply app/views/layouts/_navbar.html.erb]: done

File: app/views/layouts/application.html.erb (hunk 9 above)
Comment (STR1ND3R) [reply app/views/layouts/application.html.erb]: done

File: app/views/home/index.html.erb (hunk 7 above)
Comment (STR1ND3R) [reply app/views/home/index.html.erb]: done

File: test/controllers/home_controller_test.rb [hunk 10]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class HomeControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Comment (juperameza) [inline test/controllers/home_controller_test.rb]: If you dont need thisdelete it 
Don leave white comments
```suggestion

```

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/views/home/index.html.erb (hunk 7 above)
Comment (STR1ND3R) [reply app/views/home/index.html.erb]: done

File: test/controllers/home_controller_test.rb (hunk 10 above)
Comment (STR1ND3R) [reply test/controllers/home_controller_test.rb]: done

**Review by juperameza — APPROVED** [review]
LGTM

**Review by MauricioNC — APPROVED** [review]

**Pull request merged**